test/server/cards/06-WoE/Absolve.spec.js

```diff
@@ -0,0 +1,39 @@
+describe('Absolve', function () {
+    describe("Absolve's ability", function () {
```

Comment:
It would be interesting to add a test case where you heal more creatures than amber from your opponent and from yourself.

The engine should prompt for creatures to capture.

Reply:
Yep, there's something similar for Pandemonium. I'll add a test case 👍 .

Reply:
Added!